next/pages/cemeteries/[...fullPath].tsx

```diff
@@ -60,21 +61,26 @@ const CemeteryPage = ({ navigation, entity, general }: CemeteryPageProps) => {
                 >
                   {t('navigate')}
                 </Button>
-              )}
+              ) : null}
             </div>
           </SectionBoxed>
-          {description && (
+          {description ? (
             <SectionBoxed title={t('aboutCemetery')}>
               <RichText content={description} coloredTable={false} />
             </SectionBoxed>
-          )}
-          {general?.attributes?.cemeteryOpeningHours && (
+          ) : null}
+          {general?.attributes?.cemeteryOpeningHours ? (
             <SectionBoxed title={t('openingHours')}>
               <OpeningHours
                 openingHours={overrideOpeningHours || general?.attributes?.cemeteryOpeningHours}
               />
             </SectionBoxed>
-          )}
+          ) : null}
+          {gallery ? (
+            <SectionBoxed title={t('gallery')}>
```

Comment:
Could we use `title` field from Strapi please? It does make sense to me to let content admins to change it instead of hardcoding it (and keeping unused field in strapi). :)